lib/course.rb

```diff
@@ -1 +1,19 @@
+class Course
+  attr_reader :name, 
+              :capacity,
+              :students
 
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @students = []
+  end
+
+  def full?
+    students.count == 2
+  end
```

Comment:
This should be dynamic, checking the length of the students array compared to a the capacity:
```suggestion
  def full?
    students.count >= @capacity
  end
```